l10n_ch_payment_slip/models/payment_slip.py

```diff
@@ -132,7 +132,7 @@ def _compute_amount_hook(self):
         :return: total amount of payment slip
         :rtype: float
         """
-        return self.move_line_id.debit
+        return self.move_line_id.amount_residual
```

Comment:
I'm not really sure if this is ok or not...  :thinking: 

On one hand, it makes sense the company wants to get paid only what's owed, but on the other hand there could be use cases where the company wants their invoices to be fully paid and manage overpayments with credit notes. 

Anyway if the company works with the latter case, IMO there shouldn't have been any reconciliations, so I guess it's ok to use `amount_residual` all the time...